examples/module1/lesson1/validate-it/validation.ts

```diff
@@ -0,0 +1,26 @@
+import { INTEGER_VALIDATORS, isInteger } from './number-validation';
+
+export function validateNumber(value: string): string {
+  if (!isInteger(value)) {
+    return 'Value is empty or is not an interger.';
+  }
+  const validationResults = INTEGER_VALIDATORS.map((validator) =>
+    validator(+value)
+  );
+  const isValidNumber = validationResults.every((result) => result.isValid);
```

Comment:
Why did you introduce two separate functions? // Greetings from recording ;)

Reply:
what do you mean by two funcitons :) ?